### ISSUE TITLE
int8 question

### ISSUE THREAD

**Issue by flynnamy** [openvinotoolkit/training_extensions#153]
I do some test about resnet50_int.pth and convert it to .xml and .bin,I find that its type is float32, not int8.How to understand fakequantize?I find it uses fakequantize.Is there a way to get a model which type is int8?Looking forward to your reply!

**Comment by AlexanderDokuchaev** [comment]
@AlexKoff88

**Comment by AlexKoff88** [comment]
The current approach in OpenVINO implies to keep model with FakeQuantize operations in IR. The real transformation to INT8 happens during model inference. This approach is under development now and full support from OpenVINO side should be enabled in the next release. Please follow up.

**Comment by flynnamy** [comment]
Hi,could you please show it how to transform to INT8 in details(because it does quantization by fakequantize, but I can not find it in inference-engine) or where I can find correspoing codes?In my opinion, the difference of quantization between openvino_training_extensions and inference-engine calibration tools is that it will do finetune in openvino_training_extensions, and calibration tools does calibration in eval mode,right?

**Comment by flynnamy** [comment]
@AlexKoff88

**Comment by AlexKoff88** [comment]
You are right. NNCF (part of training extensions) does the quantization with fine-tuning (Quantization-aware Training), OpenVINO has the calibration tool which does post-training quantization with no fine-tuning. The second post-training path has better support from the Inference Engine side and actually the representation of the models after NNCF compression and calibration tool is different even in IR. 
In the next OpenVINO release, we are going to fix that and align the representation and inference pipeline for both paths.
